protocol-dashboard/scripts/updateBuild.cjs

```diff
@@ -74,8 +74,8 @@ const run = async () => {
   try {
     await updateGABuild()
     const cid = await pinGABuild()
-    const { IpfsHash } = await pinFromFs(cid)
-    fs.writeFileSync(`./build_cid.txt`, IpfsHash)
+    await pinFromFs(cid)
+    fs.writeFileSync(`./build_cid.txt`, cid.IpfsHash)
```

Comment:
this is a completely unrelated fix i'm sneaking in. same with the added header above